src/ReactiveUI.Blazor/ReactiveUI.Blazor.csproj

```diff
@@ -7,11 +7,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Reactive.Wasm" Version="1.0.1-preview.12" />
     <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.1.0" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\Users\richardb\source\repos\richbryant\Reactive.Wasm\src\System.Reactive.Wasm\System.Reactive.Wasm.csproj" />
```

Comment:
This is likely to be an issue. I would recommend just doing a NuGet reference.

Reply:
on it

Reply:
You can use NuGet wildcard versioning. Eg 1.* and then we can set the version of system.reactive.wasm to match.

Reply:
Nuts.  Okay. Will commit again.

Reply:
Build failed with wildcard versioning so I had to revert that.  Please feel free to amend this if you can make it build :(

Reply:
You should be able to wildcard now that I released 1.2.1

Reply:
it's 01:13Hrs here.  I'll do that tomorrow :) Probably mistake-prone right now.